examples/tools/epp/content/epp.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+if [ -z "$1" ]
+then
+  files=$(find ./**/*.epp)
```

Comment:
Might be worth adding `-type f` to filter out directories called `/epp`:
```suggestion
  files=$(find . -name '*.epp' -type f)
```

Reply:
Will those get caught out by the `.epp` or no?